app/backend/app.py

```diff
@@ -31,7 +32,7 @@
 # just use 'az login' locally, and managed identity when deployed on Azure). If you need to use keys, use separate AzureKeyCredential instances with the 
 # keys for each service
 # If you encounter a blocking error during a DefaultAzureCredntial resolution, you can exclude the problematic credential by using a parameter (ex. exclude_shared_token_cache_credential=True)
-azure_credential = DefaultAzureCredential()
+azure_credential = DefaultAzureCredential(exclude_shared_token_cache_credential = True)
```

Comment:
Do you want to commit 'exclude_shared_token_cache_credential = True'?

Reply:
I noticed sometimes systems get into a state where the legacy cache gets in the way. To save folks from a bunch of troubleshooting I'm leaving this set to true. If there's a more proper troubleshooting approach to this, e.g. how to locate and clear caches, etc., happy to update this.